eva_submission/eload_preparation.py

```diff
@@ -103,7 +103,7 @@ def check_submitted_filenames(self):
                        f'{", ".join(set(submitted_vcfs).difference(set(spreadsheet_vcfs)))}')
             analysis_alias = ''
             if len(eva_xls_reader.analysis) == 1:
-                analysis_alias = eva_xls_reader.analysis[0].get('Analysis Alias') or ''
+                analysis_alias = self._unique_analysis_alias(eva_xls_reader.analysis[0].get('Analysis Alias')) or ''
```

Comment:
```suggestion
                analysis_alias = self._unique_analysis_alias(eva_xls_reader.analysis[0].get('Analysis Alias', ''))
```
might be better, or maybe `_unique_analysis_alias` should check for `None` values to save us from having to do the same thing everywhere else

Reply:
Addressed in the `unique_alias`